### ISSUE TITLE
Sample Player Granulate Mode ---> legato/no retrigger option

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1186]
Playing a long sample melodically always retriggers the sample from the start.

This new grain functionality screams to have a legato and/or "no retrigger" option.
Meaning, that the sample either doesn't get retriggered when notes are overlapping or never gets retriggered at all... _(These are options in pretty good granular synthesizers like
New Sonic Arts Granite and Propellerheads Grain)_

With long stuff, you are forced to stay on one note now to avoid retrigger.
Playing melodic lines with an extremely stretched grain sample that doesn't get retriggered is pretty awesome. You should try that for yourself one time. 

_You can workaround this a little bit by automating the sample start but that's by far not 
as musical as a no retrigger option._

Best,
tL.

**Comment by kmatheussen** [comment]
There is already a way to add notes without retriggering: Add note,
move cursor down, press alt + return.
But it's not that convenient to add notes this way I guess.

**Comment by Lykdraft** [comment]
Nah. Using the glide nodes is note programming which is fine for the usual stuff where
you don't need glides all the time. But with super-long samples/grain stretched samples, where you might need non-retriggered glide all the time on every single note, playing keys on a midi keyboard, is 
much more musical.

However: Another idea would be to find a way to record **only** GLIDE notes into the track.
Can that be done somehow? (i.e.: "Record midi notes to glide node", so that the recorded note gets added the way like alt+return does? That might be a decent solution as well.)

**Comment by Lykdraft** [comment]
Another idea to solve this:

So lets say I record 3 notes likke A, C, D.
- I then let the notes glide to the next note via the  "Glide to next note at Mouse Position"
- NOW: If we could "merge" the 3 notes now into one note (the other two would then get converted into
"glide notes") that would work as well...

Might be the easiest way I can think of right now.
 (There should be an easy way to convert the data from being one thing to being another thing?)
